tests/playwright/Report.md

```diff
@@ -0,0 +1,124 @@
+# Evaluation of Microsoft Playwright
+
+## Introduction
+**Automation testing**
+Automation testing is a way to automate one's browser by simulating user actions like clicks. As Software or applications become more complex as they are built on layers and entire networks of sub-systems, including UI and API layers, external databases, networks, and even third-party integrations, there is always a need for thorough testing to be done, this ranges from unit testing to end to end testing of the application. Browser automation frameworks to perform end-to-end testing helps to ascertain the stability and the health of the application which in turn leads to the success of the application when on production. The general stability of the application doesn't entirely depend on end-to-end testing but it surely helps detects bugs, among other benefits, thus increasing application productivity. Below are some of the popular test automation frameworks;
```

Comment:
The official term is "test automation", not "automation testing". I know it's a small difference, but it's important to get the terms correct.

https://en.wikipedia.org/wiki/Test_automation

Reply:
Got it. Thanks for the correction.